README.md

```diff
@@ -5,100 +5,110 @@ Module to generate migrations based on source CSV files
 
 Main idea here is to **automatically generate migration config based on source csv**.
 
-We could use next ways of organizing source files:
-* Have next structure of folders for CSV files (like we have for default_content):
-```
--content/
---basic_page/
---article/
--taxonomy_term/
---category
---brands
--user/
-...
-```
-* Having the same, but in source filename, like described in https://github.com/davidferlay/skilld-docker-container/tree/replacedefaultcontentbymigrate/web/modules/custom/generic_content_importer#details, for example `node-basic_page.csv`, `taxonomy_term-category.csv`. etc
-
-*We could have language prefix here also, but I'd say that for 1st iteration, we shouldn't care about translation*
-
-**Main idea here - to have strict rules to identify target Entity type and Bundle for source file.**
-
 ------------
-## Fields support
+## Drush command
+
+`drush migrate_generator:generate_migrations %arg`
+
+Where %arg - absolute path to directory with source .csv files.
+
+Also we have next possible options:
+* pattern -
+  Pattern for source CSV files. Defaults to '*.csv'.
+* delimiter -
+  Delimiter for source CSV files. Defaults to ;
+* enclosure -
+  Enclosure for source CSV files. Defaults to "
+* values_delimiter -
+  Delimiter for multi-valued fields. Defaults to |
+* date_format -
+  Date format used in CSV. Defaults to "d-m-Y H:i:s"
+* file_source -
+  Type of filepath sources, used in CSV.
+  Can be "external" (means external URLs to files) or "absolute" (absolute filepath to local files).
+  Defaults to "absolute".
+
+This command creates Migration Group named `Migrate generator group` (if not exists) and migration for each source file.
+
+For now, this drush command doesn't update existing migration, so if you need to regenerate it - you'd remove existing migration fisrt.
```

Comment:
Does it mean you can't run drush command mutiple times in a row whether import is successful or not ?

Reply:
added `update` option to override previously generated migrations

Reply:
Perfect, seems to work like a charm and is enough for now